ksqldb-common/src/main/java/io/confluent/ksql/GenericKey.java

```diff
@@ -59,6 +60,7 @@ public Object get(final int index) {
     return values.get(index);
   }
 
+  @SuppressFBWarnings(value = "EI_EXPOSE_REP", justification = "values is unmodifiableList()")
   public List<?> values() {
```

Comment:
Same question as https://github.com/confluentinc/ksql/pull/7809/files#r671807853

Should we update the return type to `ImmutableList` ?